water-gui.py

```diff
@@ -106,7 +106,7 @@ def home_menu(self) -> None:
         # add an image logo
         if os.path.exists(FOLDER+"img/home.jpg"):
             img_file = ImageTk.PhotoImage(file=FOLDER+"img/home.jpg")
-            setSize(self.root, 500, 500) # WARNING: take a look for this when inserting a new image!
+            setSize(self.root, 678, 455) # WARNING: take a look for this when inserting a new image!

```

Comment:
This function `setSize()` is static, so if we change the `home.jpg` size, we must change the `setSize` parameters.